src/app/security/page.tsx

```diff
@@ -0,0 +1,119 @@
+import { Envelope } from '@phosphor-icons/react/dist/ssr'
+
+import { CardGrid } from '@/components/CardGrid'
+import { CTASection } from '@/components/CTASection'
+import { GrantsSectionCard } from '@/components/GrantsSectionCard'
+import { PageHeader } from '@/components/PageHeader'
+import { PageLayout } from '@/components/PageLayout'
+import { PageSection } from '@/components/PageSection'
+import { StructuredDataScript } from '@/components/StructuredDataScript'
+import { TextLink } from '@/components/TextLink'
+
+import { createMetadata } from '@/utils/createMetadata'
+
+import { attributes } from '@/content/pages/security.md'
+
+import { PATHS } from '@/constants/paths'
+import { FILECOIN_FOUNDATION_URLS } from '@/constants/siteMetadata'
+import { graphicsData } from '@/data/graphicsData'
+
+import { securityResourcesData } from './data/securityResourcesData'
+import { generateStructuredData } from './utils/generateStructuredData'
+
+const { header, seo } = attributes
+export const metadata = createMetadata({
+  seo,
+  path: PATHS.SECURITY.path,
+  useAbsoluteTitle: true,
+})
+
+export default function Security() {
+  return (
+    <PageLayout>
+      <StructuredDataScript structuredData={generateStructuredData(seo)} />
+      <PageHeader
+        title={header.title}
+        description={header.description}
+        image={graphicsData.security}
+        cta={{
+          href: FILECOIN_FOUNDATION_URLS.security.email.href,
+          text: FILECOIN_FOUNDATION_URLS.security.email.label,
+        }}
+      />
+
+      <PageSection
+        kicker="What We Do"
+        title="Incident Response and Monitoring"
+        description="Filecoin Foundation facilitates a collaborative approach to network security for the Filecoin network. This includes monitoring and network-wide responders, spanning multiple organizations and projects, to facilitate 24/7 monitoring of the chain across time zones ––setting the ecosystem apart in fostering a secure and resilient network. "
+      />
+      <PageSection kicker="Audits" title="Security Audits">
+        <CardGrid cols="smTwo">
+          {/* placeholder for the image */}
```

Comment:
Can we rely on the fallback?

Reply:
You mean adding a fallback image as a placeholder?

![Screenshot 2024-07-03 at 12 00 44](https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/50910606/cc7de947-b72c-484f-aa40-7e5de756f8c4)